doc/user-manual.md

```diff
@@ -1,18 +1,264 @@
-- [Metric Detail](#Metric-Detail)
-- [Metric Field](#Metric-Field)
-  - [Http Request](#Http-Request)
-  - [JDBC Statement](#JDBC-Statement)
-  - [JDBC Connection](#JDBC-Connection)
-  - [JVM Memory](#JVM-Memory)
-  - [JVM GC](#JVM-GC)
-  - [Kafka Client](#Kafka-Client)
-  - [RabbitMq Producer](#RabbitMq-Producer)
-  - [RabbitMq Consumer](#RabbitMq-Consumer)
+  
+# User Manual
 
-## Metric Detail
+- [User Manual](#user-manual)
+  - [Agent.properties](#agentproperties)
+    - [Internal HTTP Server](#internal-http-server)
+    - [Metric](#metric)
+    - [Kafka](#kafka)
+    - [HTTP Reqeust Metric](#http-reqeust-metric)
+    - [JDBC SQL Metric](#jdbc-sql-metric)
+    - [JDBC Connection Metric](#jdbc-connection-metric)
+    - [RabbitMQ Metric](#rabbitmq-metric)
+    - [Kafka Metric](#kafka-metric)
+    - [Redis Metric](#redis-metric)
+    - [JVM GC Metric](#jvm-gc-metric)
+    - [JVM Memory Metric](#jvm-memory-metric)
+    - [SQL MD5Dictionary](#sql-md5dictionary)
+    - [Tracing](#tracing)
+    - [Logging](#logging)
+  - [Prometheus Support](#prometheus-support)
+  - [Health Check and Readiness Check Endpoint](#health-check-and-readiness-check-endpoint)
+  - [Tracing](#tracing-1)
+    - [Tracing Component](#tracing-component)
+    - [Custom Span Tag](#custom-span-tag)
+      - [JDBC](#jdbc)
+      - [Cache](#cache)
+      - [RabbitMQ Producer And Consumer](#rabbitmq-producer-and-consumer)
+      - [Kafka Producer And Consumer](#kafka-producer-and-consumer)
+  - [Metric](#metric-1)
+    - [Metric Key](#metric-key)
+      - [Type And SubType Detail](#type-and-subtype-detail)
+    - [Metric Field](#metric-field)
+      - [HTTP Request](#http-request)
+      - [JDBC Statement](#jdbc-statement)
+      - [JDBC Connection](#jdbc-connection)
+      - [JVM Memory](#jvm-memory)
+      - [JVM GC](#jvm-gc)
+      - [Kafka Client](#kafka-client)
+      - [RabbitMq Producer](#rabbitmq-producer)
+      - [RabbitMq Consumer](#rabbitmq-consumer)
+  
+## Agent.properties
```

Comment:
Need a shorten text to describe what's purpose for Agent.properties

Reply:
fixed